src/components/player/YoutubePlayer.vue

```diff
@@ -55,7 +55,11 @@ export default {
             host: "https://www.youtube.com/iframe_api",
         };
 
-        const host = "https://www.youtube.com";
+        const host = (
+            this.$store.state.settings.YTEmbedVariant === "youtube"
+            // eslint-disable-next-line no-alert
+            || (this.$store.state.settings.YTEmbedVariant === "" && window.confirm(this.$t("views.settings.YTEmbedVariantPromptMsg")))
```

Comment:
only legal because vue mounts once

Reply:
Note that browsing to a multiview page can trigger this confirmation pop-up once per video, and there's no way to tell what video it is for.